apps/backend/src/data/migrations/1654703454488-create-category-modifiers-table.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey
+} from 'typeorm';
+
+export class CreateCategoryModifiersTable1654703454488
+  implements MigrationInterface
+{
+  async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'modifier-category',
+        columns: [
```

Comment:
specify for each column if it can be or cannot be null. For the possible options for a column can be found inside the typeorm column `node_modules/typeorm/schema-builder/options/TableColumnOptions.d.ts`

Reply:
Changes done